src/test/scala/ai/privado/rule/CollectionRuleTestData.scala

```diff
@@ -0,0 +1,27 @@
+package ai.privado.rule
+
+import ai.privado.model.{CatLevelOne, Constants, FilterProperty, Language, NodeType, RuleInfo}
+
+object CollectionRuleTestData {
```

Comment:
Shouldn't this be under the package `ai.privado.rule.kotlin`? Small change but will ensure better consistency, as this rule is only for kotlin.

Reply:
Ideally this should work for any language. I'll redefine the rule to make it work for any language so we don't need to move it to any any other package